tools/pylib/_boutcore_build/boutcore.pyx.jinja

```diff
@@ -13,17 +13,21 @@ import copy
 from libcpp.memory cimport unique_ptr
 from cython.operator import dereference as deref
 
+size_t = np.uint
```

Comment:
This looks a little surprising! Is it just standing in for the real `size_t`, or is it that the `indices` below do get passed as `np.uint`?

Reply:
I couldn't find the real `size_t` in cython. `np.uint` is `size_t` - at least on `x86_64` - when we later build for fedora we will verify this for some other architectures as well. `np.uint` seems to be 64-bit on 64-bit platforms, and I guess `size_t` is always unsigned, so that should cover many architectures. Luckily the compiler should raise an error if there is a mismatch, so I guess this is good enough for now.

Reply:
Any suggestion for a good comment to explain that shortly?